src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/debugger/windbg/WindbgDebugger.java

```diff
@@ -51,8 +51,8 @@ public long         readCInteger(long address, long numBytes, boolean isUnsigned
   public WindbgOopHandle readCompOopHandle(long address) throws DebuggerException;
 
   // The returned array of register contents is guaranteed to be in
-  // the same order as in the DbxDebugger for Solaris/x86 or amd64; that is,
-  // the indices match those in debugger/x86/X86ThreadContext.java or
+  // the same order as in the DbxDebugger for Solaris or amd64; that is,
+  // the indices match those in debugger/amd64/AMD64ThreadContext.java or
```

Comment:
Now you have debugger/amd64/AMD64ThreadContext.java both on this line and the next. You need to reduce it to just one occurrence.